src/components/GaugeCard/GaugeCard.jsx

```diff
@@ -0,0 +1,201 @@
+                  className={classnames(
+                    `${iotPrefix}--gauge`,
+                    { [`${iotPrefix}--gauge__loaded`]: loadedState },
+                    className
+                  )}
+                  percent="0"
+                  style={{
+                    '--gauge-value': values[gauge.dataSourceId] || 0,
+                    '--gauge-max-value': gauge.maximumValue,
+                    '--gauge-colors': color,
+                    '--gauge-bg': gauge.backgroundColor,
+                    '--stroke-dash': getStrokeDash(values[gauge.dataSourceId]) || 0,
+                    '--gauge-size': `${gaugeSize}px`,
+                    '--gauge-trend-color': gauge.trend.color,
+                  }}
+                >
+                  <circle
+                    className={`${iotPrefix}--gauge-bg`}
+                    cx={gaugeSize / 2}
+                    cy={gaugeSize / 2}
+                    r={radius}
+                  />
+                  <circle
+                    className={`${iotPrefix}--gauge-fg`}
+                    cx={gaugeSize / 2}
+                    cy={gaugeSize / 2}
+                    r={radius}
+                  />
+                  <text
+                    className={classnames(`${iotPrefix}--gauge-value`, {
+                      [`${iotPrefix}--gauge-value__centered`]: !grade,
+                    })}
+                    x={gaugeSize / 2}
+                    y="29"
+                    textAnchor="middle"
+                  >
+                    <tspan>{values[gauge.dataSourceId] + gauge.units}</tspan>
+                  </text>
+                  <text
+                    className={`${iotPrefix}--gauge-rating`}
+                    x={gaugeSize / 2}
+                    y="50"
+                    textAnchor="middle"
+                  >
+                    <tspan>{grade}</tspan>
+                  </text>
+                </svg>
+              </meter>
+              {values[gauge.trend.dataSourceId] && (
+                <div
+                  className={classnames(`${iotPrefix}--gauge-trend`, {
+                    [`${iotPrefix}--gauge-trend__up`]: gauge.trend.trend === 'up',
+                    [`${iotPrefix}--gauge-trend__down`]: gauge.trend.trend === 'down',
+                  })}
+                  key={`${gauge.trend.dataSourceId}-${i}`}
+                >
+                  <p style={{ '--gauge-trend-color': gauge.trend.color }}>
+                    {values[gauge.trend.dataSourceId]}
+                  </p>
+                </div>
+              )}
+            </React.Fragment>
+          );
+        })}
+      </div>
+    </Card>
+  );
+};
+
+GaugeCard.propTypes = {
+  ...CardPropTypes,
+  ...GaugeCardPropTypes,
+};
+
+GaugeCard.defaultProps = {
+  isLoading: false,
+  description: null,
+  content: {
+    gauges: [
+      {
+        dataSourceId: null,
+        units: '%',
+        minimumValue: 0,
+        maximumValue: 100,
+        renderValueFunction: null,
```

Comment:
I like this... but I didn't see this used anywhere.

Reply:
I wanted to get this simple implementation out to satisfy the current use case. But we have plans to extend this to accept a threshold function, different types of gauges, etc.

Reply:
I will address these changes and comments in our upcoming sprint. [Issue](https://github.com/IBM/carbon-addons-iot-react/issues/888) to track